### ISSUE TITLE
Keyboard focus broken when using Windows Emoji selector (`win+;`)

### ISSUE THREAD

**Issue by MulverineX** [feschber/lan-mouse#128]
The mouse & global hotkeys still work, but typing to search emojis over lan-mouse doesn't work, characters don't appear in the native textbox nor the search field.

**Comment by feschber** [comment]
Did this work before?

**Comment by MulverineX** [comment]
No, this wasn't a regression, I just forgot to bring this up earlier

**Comment by feschber** [comment]
Thats a weird one, but I can reproduce.

**Comment by 0Ky** [comment]
The same issue with `Win+L`, it locks only the host machine where the keyboard is physically connected, not the remote PC that's in focus.

**Comment by feschber** [comment]
@0Ky #373
